assets/icons.tsx

```diff
@@ -15,6 +16,7 @@ export type IconType =
 
 const IconSvgs: Record<IconType, React.ReactElement> = {
   home_outline: <Ionicons name="home-outline" size={23} />,
+  back_icon: <Ionicons name="chevron-back-outline" size={15} />,
```

Comment:
We can actually delete this back icon since the implementation of the back button just uses the < character.